tests/unit_tests/FixHGVS.php

```diff
@@ -41,30 +41,123 @@ public function testFixHGVS ($sInput, $sOutput)
-            // Unneeded parentheses.
-            array('g.(100_200)del', 'g.100_200del'),
+            array('g.123A>GC', 'g.123delinsGC'),
+
+            // Added bases for wildtype.
+            array('c.123T=', 'c.123='),
+            array('c.123_124TG=', 'c.123_124='),
+
+            // Floating parentheses.
+            array('g.((123_234)_(345_456)del', 'g.(123_234)_(345_456)del'),
+            array('g.(123_234)_(345_456))del', 'g.(123_234)_(345_456)del'),
+
+            // Misplaced parentheses.
+            array('(c.(123_125)insA)', 'c.((123_125)insA)'),
+
+            // Redundant parentheses.
+            array('c.1_2ins(A)', 'c.1_2insA'),
+
+            // Wrongly placed suffixes.
+            array('c.123delA', 'c.123del'),
+
+            // Wrongly formatted suffixes.
+            array('c.1_2ins[A]', 'c.1_2insA'),
+
+            // Redundant question marks.
+            array('g.?del', 'g.?del'),
+            array('g.1_?del', 'g.1_?del'),
+            array('g.?_100del', 'g.?_100del'),
+            array('g.?_?del', 'g.?del'),
+            array('g.(?_?)del', 'g.?del'),
+
+            array('g.(?_5)_10del', 'g.(?_5)_10del'),
+            array('g.(5_?)_10del', 'g.(5_?)_10del'),
+            array('g.(5_?)_?del', 'g.(5_?)del'),
+            array('g.(?_?)_10del', 'g.?_10del'),
+
+            array('g.5_(10_?)del', 'g.5_(10_?)del'),
+            array('g.5_(?_10)del', 'g.5_(?_10)del'),
+            array('g.?_(?_10)del', 'g.(?_10)del'),
+            array('g.5_(?_?)del', 'g.5_?del'),
+
+            array('g.(?_5)_(10_?)del', 'g.(?_5)_(10_?)del'),
+            array('g.(5_?)_(?_10)del', 'g.(5_?)_(?_10)del'),
+            array('g.(5_?)_(?_10)del(3)', 'g.(5_10)del(3)'),
+
+            array('g.(?_?)_(?_?)del', 'g.?del'),
+
             // Swaps positions when needed.
-            array('g.200_100dup', 'g.100_200dup'),
-            array('g.500_(100_200)del', 'g.(100_200)_500del'),
-            array('g.(400_500)_100del', 'g.100_(400_500)del'),
-            // Correct RNA-like descriptions.
-            array('c.4780delinsgagagauu', 'c.4780delinsGAGAGATT'),
+            array('g.2_1dup', 'g.1_2dup'),
+            array('g.(5_1)_10dup', 'g.(1_5)_10dup'),
+            array('g.1_(7_5)dup', 'g.1_(5_7)dup'),
+            array('g.(7_5)_1dup', 'g.1_(5_7)dup'),
+            array('c.5+1_5-1dup', 'c.5-1_5+1dup'),
+
+
+            // > UNFIXABLE VARIANTS.
+            array('g.1delinsA', 'g.1delinsA'), // Fixme; take another look.
+            array('c.1_2AC[20]', 'c.1_2AC[20]'),
+            array('c.1_2A>G', 'c.1_2A>G'),
+            array('g.=', 'g.='),
+            array('c.1insA', 'c.1insA'),
+            array('c.1_2ins', 'c.1_2ins'),
+            array('c.1_10insA', 'c.1_10insA'),
+            array('c.(1_2)insA', 'c.(1_2)insA'),
+            array('c.1_20insBLA', 'c.1_20insBLA'),
+            array('c.1_100insA', 'c.1_100insA'),
+            array('c.1_100del(10)', 'c.1_100del'), // Fixme; take another look!!
```

Comment:
This variant is listed as unfixable but is actually edited and fixed. Is it supposed to be there?

Reply:
It is added to the test to make note of the fact that this happens, but I meant to talk through this one. We can do so later today :)

Reply:
I see it's not updated (as in, `lovd_fixHGVS()` still removes the suffix). I can't find this in your Trello board, but we discussed not removing the suffix in this case, right?